src/Jellyfin.Plugin.Dlna/ContentDirectory/ControlHandler.cs

```diff
@@ -1023,18 +1011,34 @@ private QueryResult<ServerItem> GetLatest(BaseItem parent, InternalItemsQuery qu
     {
         query.OrderBy = Array.Empty<(ItemSortBy, SortOrder)>();
 
+        int limit;
+
+        if (query.StartIndex > 0)
+        {
+            limit = (query.Limit <= 0) ? int.MaxValue : (query.StartIndex.Value + (query.Limit ?? 50));
+        }
+        else
+        {
+            limit = query.Limit ?? 50;
+        }
+
         var items = _userViewManager.GetLatestItems(
             new LatestItemsQuery
             {
                 // User cannot be null here as the caller has set it
                 User = query.User!,
-                Limit = query.Limit ?? 50,
+                Limit = limit,
```

Comment:
Why not just pass `StartIndex` into this query?

Reply:
I tried, and it did nothing. Looking into Jellyfin's code, I didn't find it referencing startIndex when responding to that query, so I think it just ignores it. It seems like this is intended behavior, because if you search for "get latest" on api.jellyfin.org, there is no `startIndex` argument for `GET /Items/Latest`, but there is for `GET /Items`.